packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -642,6 +642,160 @@ describe('RichTextEditor', () => {
                 'P'
             ]);
         });
+
+        describe('Absolute link interactions in the editor', () => {
```

Comment:
Should have a test for mixed-case or uppercase link e.g. ``HTTP://foo``.

Reply:
Added